### PR TITLE
Add ADR for third-party security tools

### LINKED ISSUE #36: Decide on security analysis tools
We want to decide on security tools that we can integrate into our development and continuous integration processes. 

| Type of Check | Alex | Carly | Amy |
|----------------|-----|------|------|
| Dependency scanning | [GitHub Dependabot](https://dependabot.com/) | Snyk, but open to hearing about others!  | Snyk |
| Static analysis for security | [GitHub CodeQL](https://securitylab.github.com/tools/codeql/) | Not sure. Open to opinions | Bandit | 
| Application scanning | OWASP Zap | OWASP Zap | Not sure, Open to anything you choose |

### PR DESCRIPTION
**Pull request by alexsoble**
## What does this change? 

Resolves #36.

## What does this change look like?

[Preview link.](https://github.com/18F/OPRE-Unicorn/blob/ars/36/adr-for-security-tools/docs/adr/009-use-github-dependabot-and-codeql.md) 👁️

### CODE REVIEW

**Review by carjug — APPROVED** [review]
lgtm!

File: docs/adr/009-use-github-dependabot-and-codeql.md
Comment (amymok) [inline docs/adr/009-use-github-dependabot-and-codeql.md:37]: "Dependabot" should be "CodeQL" here

**Review by amymok — CHANGES_REQUESTED** [review]
LGTM, just one change request

**Review by amymok — APPROVED** [review]
LGTM!

**Comment by alexsoble** [comment]
@ninamak You're the last approver on this one — I believe that once this merges, we are all done with our initial round of Architecture Decision Records / tech choices! 🎉 🎉

**Review by ninamak — APPROVED** [review]
LGTM!

**Pull request merged**
